Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -158,6 +158,10 @@ The [zone](../../Storage-Configuration/Object-Storage-RGW/ceph-object-multisite.
 The hosting settings allow you to host buckets in the object store on a custom DNS name, enabling virtual-hosted-style access to buckets similar to AWS S3 (https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html).
 
 * `dnsNames`: a list of DNS names to host buckets on. These names need to valid according RFC-1123. Otherwise it will fail. Each endpoint requires wildcard support like [ingress loadbalancer](https://kubernetes.io/docs/concepts/services-networking/ingress/#hostname-wildcards). Do not include the wildcard itself in the list of hostnames (e.g., use "mystore.example.com" instead of "*.mystore.example.com"). Add all the hostnames like openshift routes otherwise access will be denied, but if the hostname does not support wild card then virtual host style won't work those hostname. By default cephobjectstore service endpoint and custom endpoints from cephobjectzone is included. The feature is supported only for Ceph v18 and later versions.
+Please consider below points while setting this option:
+  * Domains added to `dnsNames` should listen to the same port as the RGW Kubernetes service.
+  * If the RGW Kubernetes service listens to both `secure` and `non-secure` ports, include domains which use a secure port.
+  * If TLS is enabled with RGW, either include domains in `dnsNames` to the certificate or set `insecureSkipVerify` to true in the kubernetes secret containing TLS cert.
```

Comment:
I'm confused about the wording here. Could you attempt to use more clear wording?